api_address_alias.go

```diff
@@ -421,3 +421,98 @@ func (a *AddressAliasAPIService) ListAliasesExecute(r ApiListAliasesRequest) ([]
 
 	return localVarReturnValue, localVarHTTPResponse, nil
 }
+
+type ApiDeleteAliasesRequest struct {
+	ctx            context.Context
+	ApiService     *AddressAliasAPIService
+	aliasAddresses []string
+}
+
+func (r ApiDeleteAliasesRequest) Execute() (*http.Response, error) {
+	return r.ApiService.DeleteAliasesExecute(r)
+}
+
+/*
+DeleteAliases Deletes a list of users with aliases and their corresponding aliases
```

Comment:
```suggestion
DeleteAliases Deletes the corresponding aliases of users in 'aliasAddressses'
```
This endpoint no longer deletes users, only their aliases